test/functional/apps/console/_autocomplete.ts

```diff
@@ -53,7 +52,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('should provide basic auto-complete functionality', async () => {
       await PageObjects.console.enterText(`GET _search\n`);
-      await PageObjects.console.pressEnter();
```

Comment:
we are already adding a `\n` with the line above, and triggering another `enter` on the editor adds a race condition that could result in a new line with autocomplete open. Since the first option of the autocomplete for an empty line is DELETE, we could end up with the wrong text being entered in the editor.